README.md

```diff
@@ -18,6 +18,30 @@ sudo udevadm control --reload-rules && sudo udevadm trigger
 Install depthai-ros. (Available for Noetic, foxy, galactic and humble)
 `sudo apt install ros-<distro>-depthai-ros`
 
+## Docker
+You can additionally build and run docker images on your local machine. To do that, add USB rules as in above step, clone the repository and inside it run (it matters on which branch you are on):
+```
+docker build --build-arg USE_RVIZ=1 -f ./desktop.dockerfile -t depthai_ros .
```

Comment:
```suggestion
docker build --build-arg USE_RVIZ=1 -t depthai-ros .
```

Reply:
I've changed the tag name to reflect on the branch name, as when it comes to installing rviz, current launch files run it by default, so maybe we should keep it?

Reply:
I was thinking more in the lines of adding it as a "tag" on the depthai-ros name. 
So should at the end be: `luxonis/depthai-ros:${BRANCH_NAME}`

and then there is not latest (or latest is whatever the latest ROS OS base is)

Reply:
> to installing rviz, current launch files run it by default, so maybe we should keep it?

I mean, you expose it as a parameter when building locally, but you do NOT build with it in CI. 
I assume we'd want rather more complex prebuilt docker images than not, such that more things can be done out of the box.

TLDR add the build arg `USE_RVIZ` to CI when building the images as well? Or is it ON by default?

Reply:
Updated flags, modified CI file a little bit and reformatted the code, now it all lights up green :slightly_smiling_face: